components/esp32/project_include.cmake

```diff
@@ -26,6 +26,7 @@ endif()
 # Warn if the toolchain version doesn't match
 #
 # TODO: make these platform-specific for diff toolchains
-get_expected_ctng_version(expected_toolchain expected_gcc)
-gcc_version_check("${expected_gcc}")
-crosstool_version_check("${expected_toolchain}")
+# Zephyr: remove ctng, gcc and version check
+# get_expected_ctng_version(expected_toolchain expected_gcc)
+# gcc_version_check("${expected_gcc}")
+# crosstool_version_check("${expected_toolchain}")
```

Comment:
What is the need to keep the original lines commented?

Reply:
The only reason to keep original content commented is to make easy during next hal update. What do you think?

Reply:
I see, it is here to ease rebasing. My concern is that it potentially messes up the codebase in the mid/long run.